src/annotator/guest.js

```diff
@@ -306,8 +306,7 @@ export default class Guest extends Delegator {
     });
 
     this.subscribe('annotationsLoaded', annotations => {
-      this.loadDoodles(annotations.filter(this.isDoodleAnnotation));
```

Comment:
So this is now handled by `sync` instead?

Reply:
Yep - this way it happens every time an annotation is synced, not just when they are initially loaded. So it will load newly created annotations.